projects/testing-library/tests/rerender.spec.ts

```diff
@@ -48,6 +49,41 @@ test('rerenders the component with updated inputs', async () => {
 });
 
 test('rerenders the component with updated props and resets other props', async () => {
+  const firstName = 'Mark';
+  const lastName = 'Peeters';
+  const { rerender } = await render(FixtureComponent, {
+    componentInputs: {
+      firstName,
+      lastName,
+    },
+  });
+
+  expect(screen.getByText(`${firstName} ${lastName}`)).toBeInTheDocument();
+
+  const firstName2 = 'Chris';
+  await rerender({ componentInputs: { firstName: firstName2 } });
```

Comment:
It seems like this test fails while using `componentInputs` instead of `componentProperties`.
Do you think this is the expected behavior @shaman-apprentice ? 
Changes is `{}` here.

Reply:
I think the test should pass. On a first glance I don't know why it doesn't. Unfortunately, I am  busy at the moment. When you want to create an issue for it, I promise to dig into it within the next 2-3 weeks. A quick look if you find a solution faster is of course very welcome.

Reply:
I found a few minutes to deal with it. https://github.com/testing-library/angular-testing-library/pull/380 will fix it. Good catch and thanks a lot for adding this test!